lib/unit-ruby/util/resource_operations.rb

```diff
@@ -20,7 +20,7 @@ def self.included(klass)
       end
 
       module ClassMethods
-        def create(attributes)
+        def create(attributes, id: nil)
```

Comment:
If I call this with `create(foo: 1, id: 2, bar: 3)` does it correctly map everything into `attributes= { foo: 1, bar: 3}; id = 2`?

Reply:
no I don't think so, I've been calling it like `.create({foo: 1}, id: 2)`

Reply:
yeah that's a much better idea! 🏅